sqs/src/main/scala/monix.connect.sqs/SqsOp.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.sqs
+
+import java.util.concurrent.CompletableFuture
+
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.{
+  AddPermissionRequest,
+  AddPermissionResponse,
+  ChangeMessageVisibilityRequest,
+  ChangeMessageVisibilityResponse,
+  CreateQueueRequest,
+  CreateQueueResponse,
+  DeleteMessageRequest,
+  DeleteMessageResponse,
+  DeleteQueueRequest,
+  DeleteQueueResponse,
+  GetQueueUrlRequest,
+  GetQueueUrlResponse,
+  ListQueuesRequest,
+  ListQueuesResponse,
+  ReceiveMessageRequest,
+  ReceiveMessageResponse,
+  SendMessageBatchRequest,
+  SendMessageBatchResponse,
+  SendMessageRequest,
+  SendMessageResponse,
+  SqsRequest,
+  SqsResponse
+}
+
+trait SqsOp[In <: SqsRequest, Out <: SqsResponse] {
+  def execute(sqsRequest: In)(implicit client: SqsAsyncClient): CompletableFuture[Out]
+}
+
+object SqsOp {
+
+  implicit val addPermission = SqsOpFactory.build[AddPermissionRequest, AddPermissionResponse](_.addPermission(_))
```

Comment:
can we put all these `implicits` within an `Implicits` object so people will easily understand it?
importing it will look like `import  monix.connect.sqs.SqsOp.Implicits._`

Reply:
@t1707 do you agree on this change? in this way it would be same as the dynamodb connector